### ISSUE TITLE
RSSI level sensors naming 

### ISSUE THREAD

**Issue by Kaibob2** [iMicknl/ha-tahoma#513]
### Did you read the instructions?

- [X] I have read the [README](https://github.com/iMicknl/ha-tahoma/blob/master/README.md), including the [advanced debugging](https://github.com/iMicknl/ha-tahoma/blob/master/README.md#advanced) section.

### The problem

With the most recent version every RSSI sensor is named RSSi_level_x
It's impossible to tell which is which. 

![Screenshot_20210810-200128_Samsung Internet](https://user-images.githubusercontent.com/20832276/128911854-dfba7df1-3bcb-43f2-b4f6-e181b35005fb.jpg)

### What version of this integration (ha-tahoma) has the issue?

2.6.0

### What version of Home Assistant Core has the issue?

Core-2021.8.5

### Which gateway / hub do you use?

Tahoma V2 Hub

### Device model

Positionable Roller shutter with low speed

### Device type

Shutter b

**Comment by tetienne** [comment]
You have to open the details page of the entity and you will see the linked device.

**Comment by Kaibob2** [comment]
I see. Not the best way of naming, but I don't really care about the RSSI anyway as it doesn't really change because the roller shutter are quite stationary ;)

**Comment by tetienne** [comment]
I think you are right. Most of the time the main entity_id is used as prefix for the other entities. 

Here for instance it would be sensor.bedroom_cover_rssi_level

@iMicknl what do you think?

**Comment by Kaibob2** [comment]
This is what I meant :)

**Comment by iMicknl** [comment]
I was in doubt about this change as well, especially since the sensor name will be ‘sensor.rssi_level_x’… Home Assistant doesn’t have a nice best practice for this yet.

Adding the full name before or after the sensor name could be an option, however this will clutter the view as well…

Any other opinions / remarks?

**Comment by tetienne** [comment]
@iMicknl I agree it will clutter the view but it seems to be the convention. 

Z-wave JS

![image](https://user-images.githubusercontent.com/8216238/128974298-aa4e7d0b-0049-46fb-af49-d484bca76bad.png)

Shelly 

![image](https://user-images.githubusercontent.com/8216238/128974384-55a51838-629b-4005-8c56-0dde599cc0e2.jpeg)

Hue

![image](https://user-images.githubusercontent.com/8216238/128974447-09689e38-134e-4593-a9a1-3324fd10d81a.png)

HA companion 

![image](https://user-images.githubusercontent.com/8216238/128974685-7369e78f-4f4c-4ce2-9e0a-a1969a9ef83e.png)

**Comment by tetienne** [comment]
@Kaibob2 See #516 for a fix.

**Comment by Kaibob2** [comment]
I changed the code manually and restarted. Better than before. I would say it's better than before, but the sensors are still named like before. Only the friendly name changed. Is this intended behavior?

![Screenshot_20210811-100014_Samsung Internet](https://user-images.githubusercontent.com/20832276/128992674-f0ab5304-a85e-4f1b-8d54-f3c30f3a03aa.jpg)

**Comment by tetienne** [comment]
If you want to see the new entity id you have to uninstall the integration and install it again. HA won’t change the entity id if already existing, it will only change the displayed name.

**Comment by tetienne** [comment]
@Kaibob2 You can reinstall, it shouldn’t break anything. The entity id of the covers are still the same.

**Comment by tetienne** [comment]
Ah yes, you will have to wait the PR is merged within master or release in v2.6.1.

**Comment by Kaibob2** [comment]
👍  I will wait, it's not that urgent :)
